EasyLayout.Forms/EasyLayout.cs

```diff
@@ -567,112 +548,52 @@ private static int ConstantValue(Expression expr)
 
         private static bool IsConstant(Expression expr)
         {
-            if (expr.NodeType == ExpressionType.Constant)
+            switch (expr)
```

Comment:
Did anything actually change here?  In theory I like adopting new C# features, but particularly since there aren't a lot of unit tests I lack confidence that this executes _exactly_ the same as it did before.  Can we perhaps move all the rewriting of stuff out into a separate PR?